### PR TITLE
windows_ad_join - add newname property 

### PR DESCRIPTION
**Pull request by derekgroh**
### Description

Documents the newname property to allow changing the workstation name at the time of join.

### Issues Resolved

Chef Community #windows question September 6th

### Check List

- [X] Spell Check
- [X] Local build
- [X] Examine the local build
- [X] All tests pass

### CODE REVIEW

**Comment by derekgroh** [comment]
Linked with https://github.com/chef/chef/pull/7637

**Comment by tas50** [comment]
new_name is now new_hostname so we'll want to update this PR

**Comment by derekgroh** [comment]
@tas50 7637 doesn't look like it got updated before merge, do I need to do that or will someone on your end?

**Comment by tas50** [comment]
@derekgroh I changed it on chef/chef master.

File: chef_master/source/resource_windows_ad_join.rst [hunk 1]
```diff
@@ -60,6 +61,11 @@ Properties
 
    Required. The domain user that will be used to join the domain.
 
+``new_hostname``
+  **Ruby Type:** String
+
+  "Specifies a new name for the computer in the new domain."
+
```
Comment (tas50) [inline chef_master/source/resource_windows_ad_join.rst:74]: On a new line here you'll want to say "New in Chef Client 14.5."

Comment (derekgroh) [reply chef_master/source/resource_windows_ad_join.rst:74]: Other examples I found had it at the end, let me know if it should be changed.

**Review by tas50 — APPROVED** [review]

**Comment by tas50** [comment]
@mjingle This one is good to merge

**Comment by mjingle** [comment]
@derekgroh Thank you for your documentation! We had to include your work to chef-web-docs through other means. Please see https://github.com/chef/chef-web-docs/pull/1683

**Pull request closed without merging**